javaparser-core/src/main/java/com/github/javaparser/StaticJavaParser.java

```diff
@@ -528,6 +533,23 @@ public static TypeParameter parseTypeParameter(@NotNull String typeParameter) {
      */
     public static MethodDeclaration parseMethodDeclaration(@NotNull String methodDeclaration) {
         Preconditions.checkNotNull(methodDeclaration, "Parameter methodDeclaration can't be null.");
-        return handleResult(newParser().parseMethodDeclaration(methodDeclaration));
+        return QuickJavaParser.create(getParserConfiguration()).parseMethodDeclaration(methodDeclaration);
+    }
+
+    // Private methods
+
+    private static JavaParser newParser() {
+        return new JavaParser(getConfiguration());
+    }
+
+    private static <T extends Node> T handleResult(ParseResult<T> result) {
```

Comment:
It seems to me that this method is no longer used.

Reply:
At this point, method handleResult is only used by Deprecated methods in order to provide retro-compatibility.
https://github.com/javaparser/javaparser/blob/e545c2a11c062096b96284b3a263184eb1cbe7d8/javaparser-core/src/main/java/com/github/javaparser/StaticJavaParser.java#L101-L116
But later, when this deprecated methods have been removed, this function can also be removed .)